src/Shared/CommonAssemblyInfo.cs

```diff
@@ -14,6 +14,9 @@
 [assembly: CLSCompliant(false)]
 [assembly: NeutralResourcesLanguage("en-US", UltimateResourceFallbackLocation.MainAssembly)]
 
+// This reference necessary for Multitool.
+[assembly: InternalsVisibleTo("Sarif.Multitool, PublicKey=0024000004800000940000000602000000240000525341310004000001000100433fbf156abe9718142bdbd48a440e779a1b708fd21486ee0ae536f4c548edf8a7185c1e3ac89ceef76c15b8cc2497906798779a59402f9b9e27281fb15e7111566cdc9a9f8326301d45320623c5222089cf4d0013f365ae729fb0a9c9d15138042825cd511a0f3d4887a7b92f4c2749f81b410813d297b73244cf64995effb1")]
```

Comment:
>Multitool [](start = 37, length = 9)

Could you explain why you need this? What is the exact error you get if you don't provide it?

Reply:
since CompositionUtilities is internal, we need this

---
In reply to: [481265173](https://github.com/microsoft/sarif-sdk/pull/2052#discussion_r481265173) [](ancestors = 481265173)

Reply:
Let's make `CompositionUtilities` public.

If we kept it internal -- which I don't recommend -- I wouldn't put the `InternalsVisibleTo` in the _shared_ CommonAssemblyInfo.cs. I would just add a new AssemblyInfo.cs to the driver project so that only the Driver made its internals visible to the Multitool. And I would add a more specific comment like "Multitool requires access to the internal class CompositionUtilities."

---
In reply to: [481273990](https://github.com/microsoft/sarif-sdk/pull/2052#discussion_r481273990) [](ancestors = 481273990,481265173)